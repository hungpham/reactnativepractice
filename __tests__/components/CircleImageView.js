import React from 'react';
import renderer from 'react-test-renderer';
import CircleImageView from 'components/CircleImageView/CircleImageView';

it('renders correctly', () => {
  const tree = renderer.create(
		<CircleImageView
			height={120}
			imagelink={require('assets/images/check_red.png')}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
