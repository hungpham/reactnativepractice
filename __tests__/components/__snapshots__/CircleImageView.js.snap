// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": 45,
        "height": 90,
        "justifyContent": "center",
        "width": 90,
      },
      Object {
        "borderRadius": 60,
        "height": 120,
        "width": 120,
      },
    ]
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  />
</View>
`;
